aiohttp_pydantic/view.py

```diff
@@ -24,28 +24,42 @@ class PydanticView(AbstractView):
     An AIOHTTP View that validate request using function annotations.
     """
 
+    # Allowed HTTP methods; overridden when subclassed.
+    allowed_methods: Set[str] = {}
+
     async def _iter(self) -> StreamResponse:
-        method = getattr(self, self.request.method.lower(), None)
-        resp = await method()
-        return resp
+        if (method_name := self.request.method) not in self.allowed_methods:
```

Comment:
For me, we can remove `_iter` and `_raise_allowed_methods` method because they are already defined in the parent class.